src/main/scala/edu/ie3/simona/model/em/EmModelShell.scala

```diff
@@ -63,16 +62,17 @@ final case class EmModelShell(
         assetInput -> updatedFlexOptions
     }
 
-    aggregateFlex.aggregateFlexOptions(updatedAllFlexOptions)
+    val aggregate = aggregateFlex.aggregateFlexOptions(updatedAllFlexOptions)
+    (aggregate.ref, aggregate.min, aggregate.max)
```

Comment:
You changed the method `aggregateFlexOptions` in `EmAggregateFlex` to return `MinMaxFlexOptions`. Therefore, shouldn't this method also return `MinMaxFlexOptions`?

Reply:
I was hesitating a bit here, because eventually I want to get some abstractions into here. But for now, I did what you suggested